setup.py

```
@@ -66,6 +67,7 @@ def get_version():
     entry_points={
         'neon.plugin.solver': PLUGIN_ENTRY_POINT,
         "opm.transformer.dialog": DIALOG_PLUGIN_ENTRY_POINT,
+        'opm.solver.summarization': SUMMARIZER_ENTRY_POINT,
```

Ruff (0.8.2)

Undefined name `SUMMARIZER_ENTRY_POINT`

(F821)